packages/uploadthing/test/__test-helpers.ts

```diff
@@ -74,6 +73,9 @@ export const mockExternalRequests: FetchEsque = async (url, init) => {
     if (url.pathname === "/api/requestFileAccess") {
       return Response.json({ url: "https://example.com" });
     }
+    if (url.pathname === "/api/serverCallback") {
+      return Response.json({ status: "ok" });
```

Comment:
should depend on method, right?

Reply:
yea - don't have any client test so we never use the GET method in tests yet ;)